### ISSUE TITLE
_datagrid_name should be uniqueId of component

### ISSUE THREAD

**Issue by martinknor** [contributte/datagrid#753]
Because  `$.nette.ext('datagrid.reset-filter-by-column'` works with name, but if component is under other component - link to reset column doesnt work.

For example, i have component **test** and **grid** under test component. link to reset filter is `&do=test-grid-resetFilter`, but name is only grid and nette.ext would change `&do=grid-resetFilter`

`new_href = href.replace('do=' + payload._datagrid_name + '-resetFilter', 'do=' + payload._datagrid_name + '-resetColumnFilter');`

**Comment by Sofiosko** [comment]
I Agree, can you make a pull request? I do not know if owner maintains this library but this is easy fix.
